### PR TITLE
Chall03 solution fvega-tr

### PR DESCRIPTION
**Pull request by fvega-tr**

### CODE REVIEW

File: chall03/fvega-tr.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```
Comment (anna-dominguez) [inline chall03/fvega-tr.py:1]: Good

File: chall03/fvega-tr.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+					
+					else:
+						break
+
+		print(draw)
+
+	else:
+		print("usage: %s <1-9 squared_rows...>" % sys.argv[0])
+
+except:
```
Comment (anna-dominguez) [inline chall03/fvega-tr.py:86]: Bad

File: chall03/fvega-tr.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+	print("usage: %s <1-9 squared_rows...>" % sys.argv[0])
```
Comment (anna-dominguez) [inline chall03/fvega-tr.py:87]: Worst, you have your error usage in your `try` and in your `exception`

File: chall03/fvega-tr.py (hunk 2 above)
Comment (anna-dominguez) [reply chall03/fvega-tr.py:86]: A `try`/`catch` must be done for one specific thing, not for all you program. It's a very bad habit to surround your code with those.

**Review by anna-dominguez — COMMENTED** [review]
You didn't forgot the shebang but we asked for an executable this time too, you didn't set the rights of your file and we cannot test it. You should read the `README` of the challenge

**Comment by anna-dominguez** [comment]
Not an executable and no comment of your work 😞

**Comment by fvega-tr** [comment]
Ok, I will try to improve for next time, thank you very much for the
correction.
I thought about the permissions later, I did not remember that I was
working on windows.

**Pull request closed without merging**